{{cookiecutter.project_slug}}/terraform/control_plane/README.md

```diff
@@ -0,0 +1,8 @@
+# Controlplane
+
+ArgoCD and SealedSecrets are configured on the SFU controlplane.
+If you are not using the SFU controlplane you can set up your own controlplane on the EC2 instance using the makefile command
```

Comment:
For the cookie cutter, we should assume it is installed in the same cluster as the application.